samples/excel/01-basics/basic-api-call.yaml

```diff
@@ -32,7 +32,6 @@ script:
                 OfficeHelpers.Utilities.log(error);
             }
         }
-
```

Comment:
Actually, it was me who added these :-)  But that's fine, doesn't really matter whether they're there or not.